WalletWasabi/KeyManagement/KeyManager.cs

```diff
@@ -65,14 +65,13 @@ public class KeyManager
 		// BIP84-ish derivation scheme
 		// m / purpose' / coin_type' / account' / change / address_index
 		// https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki
-		public static readonly KeyPath DefaultAccountKeyPath = new KeyPath("m/84'/0'/0'");
+		public static readonly KeyPath DefaultAccountKeyPath = new KeyPath("m/84h/0h/0h");
```

Comment:
Just curious, why is this?

Reply:
To use it consistently in the same form. At other places the `'` symbol is problematic when sending it to HWI.